cirq-core/cirq/transformers/routing/route_circuit_cqc.py

```diff
@@ -25,30 +25,28 @@
 if TYPE_CHECKING:
     import cirq
 
-QidPair = Tuple['cirq.Qid', 'cirq.Qid']
+QidIntPair = Tuple[int, int]
 
 
-def disjoint_nc2_combinations(qubit_pairs: List[QidPair]) -> List[Tuple[QidPair, ...]]:
+def disjoint_nc2_combinations(
+    qubit_pairs: Sequence[QidIntPair],
```

Comment:
This seems like a less convenient argument type than `QidPair`s. If this method isn't expected to be used directly, that's fine, but if humans are expected to call this then it may be preferable to convert qubits to ints within the method instead.

Reply:
This function is not supposed to be called by users. It's an internal implementation detail. Made the function private.